packages/static-build/test/build.test.ts

```diff
@@ -1,54 +1,88 @@
 import path from 'path';
+import { remove } from 'fs-extra';
 import { build } from '../src';
 
 describe('build()', () => {
-  it('should detect Builder Output v3', async () => {
```

Comment:
What happened to `should detect Builder Output v3`?

Reply:
It's still there, moved down below.